examples/lib/main.dart

```diff
@@ -19,9 +23,29 @@ import 'package:examples/stories/system/system.dart';
 import 'package:examples/stories/tiled/tiled.dart';
 import 'package:examples/stories/utils/utils.dart';
 import 'package:examples/stories/widgets/widgets.dart';
+import 'package:flame/game.dart';
 import 'package:flutter/material.dart';
 
 void main() {
+  var page = window.location.search ?? '';
```

Comment:
I'm trying to make embedded `flutter-app` examples work. Not sure if it helps though, I don't know how to test it locally.

Reply:
See the documentation here: https://docs.flame-engine.org/main/development/documentation.html#building-documentation-locally

Reply:
Just realized that this breaks the examples app for all platforms except web, so we'll have to come up with a way to be able to run this on all platforms.

Reply:
But for all other platforms `window.location.search` would be null, so it will be run as dashbook? Or am I missing something?

Reply:
The use of dart:html will not allow it to compile to other platforms than web

Reply:
Maybe something like this could work instead of pulling in a new package?
https://gpalma.pt/blog/conditional-importing/

Reply:
Please check it out https://github.com/flame-engine/flame/pull/2411